packages/ra-supabase-ui-materialui/README.md

```diff
@@ -2,34 +2,34 @@
 
 This package provides components to integrate [Supabase](https://supabase.io/) with [react-admin](https://marmelab.com/react-admin) when using its default UI ([ra-ui-materialui](https://github.com/marmelab/react-admin/tree/master/packages/ra-ui-materialui)).
 
-In particular, this package provides components around Supabase authentication with the following workflow:
+## Installation
 
-1. You invite users from the Supabase Admin page.
-2. Users use the invite link they received by email.
-3. They arrive on a page where they can set their password.
-4. They can now login using their email and password.
+```sh
+yarn add ra-supabase-ui-materialui
+# or
+npm install ra-supabase-ui-materialui
+```
 
 ## Usage
 
 ```jsx
 // in supabase.js
 import { createClient } from '@supabase/supabase-js';
 
-export const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY);
+export const supabase = createClient('YOUR_SUPABASE_URL', 'YOUR_SUPABASE_ANON_KEY');
 
 // in dataProvider.js
-import { supabaseDataProvider } from 'ra-supabase';
-import { supabase } from './supabase';
-
-const resources = {
-    posts: ['id', 'title', 'body', 'author_id', 'date'],
-    authors: ['id', 'full_name'],
-};
+import { supabaseDataProvider } from 'ra-supabase-core';
+import { supabaseClient } from './supabase';
 
-export const dataProvider = supabaseDataProvider(supabase, resources);
+export const dataProvider = supabaseDataProvider({
+    instanceUrl: 'YOUR_SUPABASE_URL',
+    apiKey: 'YOUR_SUPABASE_ANON_KEY',
```

Comment:
Do you really need to provide `instanceUrl` and `apiKey` again here?

Reply:
Yes we do. It's necessary for authenticated requests